src/bit-systems/object-menu.ts

```diff
@@ -39,10 +54,69 @@ function moveToTarget(world: HubsWorld, menu: EntityID) {
   const targetObj = world.eid2obj.get(ObjectMenu.targetRef[menu])!;
   targetObj.updateMatrices();
 
+  // TODO: position the menu more carefully...
```

Comment:
> We can do it in follow-up PR.

Yes, that's the plan.

Reply:
We should probably just aim for roughly what we have now in terms of menu positioning, we still would like to ultimately not have these in world object menus in their current form on most platforms. On desktop and mobile we likely end up with more traditional 2d context menus, and in VR we likely don't need most of what the menus do since direct manipulation is perdurable (we still need to figure out what to do with the rest of the options)